html.md

```diff
@@ -64,23 +64,25 @@
 ## 3. Buttons
   > Buttons can submit a form, be a link styled like a button, or simply just a button.
 
+Buttons should always have the `type` attribute set to `button` unless it is inside a `form` element. Without this an unexpected form submit will happen if it is clicked on a page with a form since the default is `type=submit`.
+
 ### 3.1 Submit Buttons
 
-   When a button is needed to submit a form, ``<button>`` should be the tag used with a ``type='submit'``.
+   When a button is needed to submit a form, `<button>` should be the tag used with a `type='submit'`.
 
 ```html
 <button class='primaryButton' type='submit'>Submit</button>
 ```
 ### 3.2 Links
-  For a link to another page just needs to be styled like a button, use the ``<a>`` tag like you normally would.
+  For a link to another page just needs to be styled like a button, use the `<a>` tag like you normally would.
```

Comment:
Sounds weird - how about: `When a link to another page needs to be styled like a button,`